circuitbreaker/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: circuitbreaker_web_1
+      APP_PORT: 9235
+      
+  web:
+    image: ghcr.io/lightningequipment/circuitbreaker:v0.4.0@sha256:27d1a70ecacb15ff827ea1eca1fee39b7761d3ee602cdb169aec339e756daa98
```

Comment:
I've tagged v0.4.1 which contains the icon as favicon.

Reply:
Excellent, I will update the PR.